pages/about.html

```diff
@@ -2,11 +2,53 @@
 <html lang="en">
 <head>
   <meta charset="UTF-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <link rel="stylesheet" href="../css/aboutStyle.css"/>
+  <title>Yanxian Zhang</title>
 </head>
 <body>
-  
+  <div class="headMessage">
+    <nav class="menu">
+      <ul>
+        <a href="index.html" target="_top"> Home </a>
+        <a href="about.html" target="_top"> ABOUT </a>
+        <a href="portfolio.html" target="_top"> Protfolio </a>
+      </ul>
+    </nav>
+  </div>
+
+  <div class="outer_container">
+    <div class="inner_container">
+      <div class="past"> 
+        <h1> The Past</h1>
+        <img class="surrounding1" src="../assets/past.jpeg" alt="miranda-selfie" width="35%">
```

Comment:
The width attribute of the `img` tag is in pixels. You cannot use the `%` symbol, it causes errors.